City-of-Helsinki/employment-services-ui
# THF-524: add redirect to translation not exists, lang code and field …

- When content doesn't have translation Next will create link to the the local front page. Previously next created link based on the original link even the link didn't exist.
- If someone puts a link to the main menu and the link doesn't have translation the link will be visible only on language it exists.  

How to test:

- Create new basic page with any language. Don't create translations.
- Try to change language with language switcher.
- You should end up on the front page in the language you tried to view the page in.
- Link the created page to the main menu as a dropdown object
- Test that the link is only visible when the language of the page and the language you use match.
- Go back to the page edit and change the linking of the page to be top level of the menu.
- The link on the nav should be only visible when the language of the page and the language you use match.
